### PR TITLE
[FIX] open_academy: pre-commit suggested lints T#53764

### PR DESCRIPTION
**Pull request by payen000**
Split lines that were too long in session.py, corrected tab indentation
in partner.py, created readme folder and files, modified manifest to fit
current standards and deleted trailing whitespaces after some imports.

### CODE REVIEW

**Comment by payen000** [comment]
@frahikLV Code rated 10/10 both in mandatory and optional checks after a few changes! Would you review my code? :) 

(I think I love pre-commit, it gives me so much peace of mind to have a tool that tells me if I left trailing whitespaces)

File: open_academy/models/session.py [hunk 1]
```diff
@@ -10,20 +10,24 @@ class Session(models.Model):
     active = fields.Boolean(default=True)
     duration = fields.Float()
     number_of_seats = fields.Integer()
-    instructor_id = fields.Many2one('res.partner', domain="['|',('is_instructor', '=', True),('category_id.name', 'like', 'Teacher')]")
+    instructor_id = fields.Many2one('res.partner', domain="['|',"
+                                    "('is_instructor', '=', True),"
+                                    "('category_id.name', 'like', 'Teacher')]")
```
Comment (frahikLV) [inline open_academy/models/session.py:15]: This is more readable

```suggestion
instructor_id = fields.Many2one(
        "res.partner", domain="['|', ('is_instructor', '=', True), ('category_id.name', 'like', 'Teacher')]"
    )
```

File: open_academy/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+This is the module description
```
Comment (frahikLV) [inline open_academy/readme/DESCRIPTION.rst:1]: Please be more descriptive, ie.

```suggestion
Open academy
============

This is a training module; it has been created following the guidelines established in https://www.odoo.com/documentation/15.0/developer/howtos/backend.html

Author: name <vauxoo_email>
```

**Review by frahikLV — CHANGES_REQUESTED** [review]
@payen000 can you address these changes?

**Comment by payen000** [comment]
@frahikLV  done! :) would you review my code again, please?

**Review by frahikLV — APPROVED** [review]
@payen000 Thanks I will merge it :grin:

**Pull request merged**